packages/app-root/src/app/users/[login]/page.js

```diff
@@ -0,0 +1,7 @@
+export default function UserPage({ params }) {
+  return (
+    <div>
+      <p>This is {params.login}'s profile page.</p>
```

Comment:
`eslint` doesn't like the apostrophe in this paragraph, and throws an error during `yarn build` of app-root locally. I suggest using `&apos;` instead.